pkg/datapath/loader/verifier_load_test.go

```diff
@@ -26,6 +26,7 @@ func hostLoadPermutations() iter.Seq[*config.BPFHost] {
 		for permutation := range permute(1) {
```

Comment:
```suggestion
		for permutation := range permute(2) {
```

I think you'll need to do this since your are now permuting two boolean values. Accessing permutation[1] without this change should also be index out of bounds if you'd run this test locally.

Reply:
corrected.